staging/src/k8s.io/apiserver/pkg/endpoints/filters/authentication.go

```diff
@@ -92,3 +91,15 @@ func audiencesAreAcceptable(apiAuds, responseAudiences authenticator.Audiences)
 
 	return len(apiAuds.Intersect(responseAudiences)) > 0
 }
+
+func shortRequestInfo(req *http.Request) string {
```

Comment:
this info (remote address, url path, and user agent) is already included in the httplog... doing `curl -k https://localhost:6443 -H "Authorization: Bearer foo"` logs this:

```
E0811 10:50:25.993001   59995 authentication.go:53] Unable to authenticate the request due to an error: invalid bearer token
I0811 10:50:25.993299   59995 httplog.go:89] "HTTP" verb="GET" URI="/" latency="501.505µs" userAgent="curl/7.64.1" srcIP="127.0.0.1:62602" resp=401
```

the logged 401 response seems sufficient to identify the problematic request

Reply:
This only seems to be the case for loglevels >= 3 - based on
https://github.com/kubernetes/kubernetes/blob/112dbd55860e600af525cedc255f2664e3f286aa/staging/src/k8s.io/apiserver/pkg/server/httplog/httplog.go#L88-L90

That means that for log levels up until 2 you get get your logs filled with messages like:
```
E0526 11:10:32.896176       1 authentication.go:53] Unable to authenticate the request due to an error: [invalid bearer token, context canceled]
E0526 11:11:12.842792       1 authentication.go:53] Unable to authenticate the request due to an error: [invalid bearer token, context canceled]
```
that aren't really actionable.

Asking a 3rd party for logs that are more verbose is an unnecessary roundtrip when you could include this basic information within this message already.

Reply:
constructing ad-hoc alternate request attribute serialization in specific filters (in ways that will duplicate existing logs when running with v>=3) doesn't seem great. If we want to log more request details specifically in 401 scenarios, would it make more sense to adjust the http logging conditions to incorporate that?